sequencing/ATACSeq/config/config.txt

```diff
@@ -44,42 +35,67 @@ export ALIGNED_DIR="${MAIN_DIR}/3_aligned"
-export PIPFILE="${MAIN_DIR}/packagesPip.txt"
-export CONDA=/lustre/home/${USER}/.conda/envs/${PROJECT}
-export CONDA_ENVDIR="/path/to/conda/etc/folder"
-export PIP_ENV="/path/to/pip/environment/${PROJECT}"
-
-## Reference data filepaths
-export REFGENOME="/path/to/reference/genome"
-export BLACKLIST="/path/to/blacklisted/peak/regions/.gz"
-export XCHRBED="/path/to/X/CHR/genes/.bed"
-export PAR="/path/to/pseudoautosomal/regions/.bed"
-export GENOMEFASTA="/path/to/GRCh38/Sequence/WholeGenomeFasta/"
-export KGREF="/path/to/reference/100G/"
-export SNP="${GENODIR}/path/to/SNP/file/.vcf.gz"
-export UCSCRef="/path/to/UCSC/references"
-export GENOMESIZE="/path/to/genome.info"
-export SEGREFGENOME="/path/to/WholeGenome/genome.char.fa/"
-export GWASTRAITS="/path/to/gwas_traits/"
-export LDREFPATH="/path/to/LDScore/resources/"
-
-## Software filepaths
-export VERIFYBAMID="/path/to/verifyBamID_1.1.3/verifyBamID/bin/verifyBamID"
+
+## =========================================== ##
+|| Scripts and software filepaths and versions ||
+## =========================================== ##
+
+## Specify full path to pipeline's scripts directory ##
+export SCRIPTS_DIR="/path/to/this/scripts/folder"
+
+export SUB_SCRIPTS_DIR="${SCRIPTS_DIR}/subscripts"
+export RSCRIPTS_DIR="${SCRIPTS_DIR}/Rscripts"
+export LOG_DIR="${SCRIPTS_DIR}/logFiles"
+
+## Specify the version of the modules used through the pipeline. Note the minimum version required. ##
+export RVERS="R version should be > 4.2"
+export PVERS="Python version should be > 3.9"
+export ACVERS="Anaconda3 version should be > 2020.05"
+
+## Specify full path to the required softwares ##
+export RLIB="/path/to/users/R/libraries"
+export VERIFYBAMID="/path/to/verifyBamID/bin/verifyBamID,  version should be > verifyBamID_1.1.3"
 export PICARD="/path/to/picard/Java/file"
 export LDPATH="/path/to/ldsc"
 export RSTUDIO_PANDOC="/usr/lib/rstudio-server/bin/pandoc"
 
-## Adapter sequences to trim
+## Specify full path to conda environment ##
+export CONDA="/path/tp/.conda/envs/folder"
+export CONDA_ENVDIR="/path/to/conda/etc/folder"
+## Specify full path to pip environment ##
+export PIP_ENV="/path/to/pip/environment/project-name"
+
+
+## ========================= ##
+|| Sequencing data filepaths ||
+## ========================= ##
+
+## Specify full path to references folder ##
+export REFERENCES_DIR="/path/to/references/folder"
+
+## Specify the path to each of the reference datasets, files or folders ##
+export REFGENOME="${REFERENCES_DIR}/reference_genome"
+export BLACKLIST="${REFERENCES_DIR}/blacklisted_peak_regions"
+export XCHRBED="${REFERENCES_DIR}/X_chr_genes"
+export PAR="${REFERENCES_DIR}/pseudoautosomal_regions"
+export GENOMEFASTA="${REFERENCES_DIR}/Whole_genome_fasta/"
+export KGREF="${REFERENCES_DIR}/reference_1000G/"
+export SNP="${REFERENCES_DIR}/SNP_files"
+export UCSCRef="${REFERENCES_DIR}/UCSC/"
+export GENOMESIZE="${REFERENCES_DIR}/genome.info"
+export SEGREFGENOME="${REFERENCES_DIR}/genome.char.fa/"
```

Comment:
Is this a file or a directory?
Only asking because .fa is a file extension (FASTA)

Reply:
Yes, that is a file, the / is an error